packages/react-router/README.md

```diff
@@ -0,0 +1,378 @@
+# `@shopify/react-router`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt) [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-router.svg)](https://badge.fury.io/js/%40shopify%2Freact-router)
+
+A universal router for React with first-class support for the [WHATWG `URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL) and prefetching.
+
+## Installation
+
+```bash
+$ yarn add @shopify/react-router
+```
+
+## Usage
+
+### `<Router />`
+
+Creates a router that is provided to the rest of the React tree. This router powers the `useCurrentUrl()` and `useRouter()` hooks and, by extension, the rest of the features of this library. On the client, you should render a `Router` with no props. On the server, you should pass in the `url` prop, which is a `URL` object that represents the current location to simulation.
+
+```tsx
+// Example in Koa...
+
+const app = new Koa();
+app.use((ctx) => {
+  ctx.body = renderToString(
+    <Router url={ctx.URL}>
+      <App />
+    </Router>,
+  );
+});
+```
+
+This component basically just composes the `Prefetcher` component, serialization for the `NoMatch` component, alongside a number of context providers (`RouterContext`, `CurrentUrlContext`). If you wish to omit some features from your final bundle, you can skip the `Router` component and render those primitives instead.
```

Comment:
Might be nice to see what the equivalent code would look like to compose them

Reply:
It's the entire contents of the `Router` component file, do you think it's worth duplicating?

Reply:
Maybe just a link then :/